src/test/datascience/notebook.functional.test.ts

```diff
@@ -1389,20 +1386,6 @@ plt.show()`,
                 }
             });
 
-            test('Notebook launch with PYTHONWARNINGS', async function () {
```

Comment:
Shouldn't this test still pass? I believe the original intent of this was to not have PYTHONWARNINGS cause a kernel to fail to start.

Reply:
Oh shoot yes.